python/cugraph/graph/c_graph.pyx

```diff
@@ -465,19 +465,30 @@ class Graph:
     def _degree(self, x = 0):
         cdef uintptr_t graph = self.graph_ptr
         cdef gdf_graph* g = < gdf_graph *> graph
-        err = gdf_add_adj_list(g)
-        cudf.bindings.cudf_cpp.check_gdf_error(err)
 
-        n = g.adjList.offsets.size - 1
+        cdef gdf_adj_list* adj_ptr
+
+        if(g.adjList):
+            n = g.adjList.offsets.size - 1
```

Comment:
Better use num_vertices(). We also need to update num_vertices() to check g.transposedAdjList if g.adjList is nullptr.

Reply:
modified num_vertices() for checking transposed adj List